### PR TITLE
Feat: Workspace creation wizard

### PR DESCRIPTION
**Pull request by Mikehrn**
WIP; dont merge

### CODE REVIEW

File: packages/frontend-2/components/workspace/wizard/step/Pricing.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import { startCase } from 'lodash'
+
+const { state, goToNextStep, goToPreviousStep } = useWorkspacesWizard()
+const mixpanel = useMixpanel()
+
+const plans = ref(pricingPlansConfig.plans)
+const isYearlySelected = ref(false)
+
+const onCtaClick = (plan: WorkspacePlans) => {
+  state.value.plan = plan as unknown as PaidWorkspacePlans
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/wizard/step/Pricing.vue:59]: Something that can be improved in the future.

File: packages/frontend-2/lib/workspaces/composables/wizard.ts [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    }
+
+    if (state.invites.length > 0) {
+      const inputs: WorkspaceInviteCreateInput[] = state.invites.map((email) => ({
+        role: mapMainRoleToGqlWorkspaceRole(Roles.Workspace.Member),
+        email,
+        serverRole: mapServerRoleToGqlServerRole(Roles.Server.User)
+      }))
+
+      await inviteToWorkspace({ workspaceId, inputs, hideNotications: true })
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/lib/workspaces/composables/wizard.ts:179]: hideNotications > hideNotifications

File: packages/frontend-2/lib/workspaces/composables/management.ts [hunk 3]
```diff
@@ -43,7 +43,13 @@ export const useInviteUserToWorkspace = () => {
   const { mutate } = useMutation(inviteToWorkspaceMutation)
   const isWorkspacesEnabled = useIsWorkspacesEnabled()
 
-  return async (workspaceId: string, inputs: WorkspaceInviteCreateInput[]) => {
+  return async (args: {
+    workspaceId: string
+    inputs: WorkspaceInviteCreateInput[]
+    hideNotications?: boolean
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/lib/workspaces/composables/management.ts:49]: hideNotications > hideNotifications

File: packages/frontend-2/lib/workspaces/composables/management.ts [hunk 4]
```diff
@@ -85,7 +91,7 @@ export const useInviteUserToWorkspace = () => {
         }
       ).catch(convertThrowIntoFetchResult)) || {}
 
-    if (!data?.workspaceMutations.invites.batchCreate.id) {
+    if (!data?.workspaceMutations.invites.batchCreate.id && !hideNotications) {
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/lib/workspaces/composables/management.ts:94]: hideNotications > hideNotifications

File: packages/frontend-2/components/workspace/wizard/step/Details.vue [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        name="name"
+        label="Workspace name"
+        color="foundation"
+        :rules="[isRequired, isStringOfLength({ maxLength: 512 })]"
+        show-label
+        auto-focus
+        size="lg"
+        @update:model-value="updateShortId"
+      />
+      <FormTextInput
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/wizard/step/Details.vue:19]: This was maybe already there, but there is a hydration mismatch on this:
image

File: packages/frontend-2/components/workspace/wizard/step/Pricing.vue [hunk 6]
```diff
@@ -0,0 +1,83 @@
+<template>
+  <WorkspaceWizardStep title="Which plan would you like?">
+    <div class="flex flex-col max-w-5xl w-full items-center">
+      <div class="grid lg:grid lg:grid-cols-3 gap-y-2 gap-x-2 w-full">
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/wizard/step/Pricing.vue:4]: Very minor - but don't need lg:grid

File: packages/frontend-2/lib/projects/graphql/queries.ts [hunk 7]
```diff
@@ -257,7 +257,6 @@ export const projectAutomationsTabQuery = graphql(`
       }
       ...FormSelectProjects_Project
     }
-    ...ProjectPageAutomationsEmptyState_Query
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/lib/projects/graphql/queries.ts:260]: Just confirming this, I can't see where the fragment was removed in this branch.

Comment (Mikehrn) [reply packages/frontend-2/lib/projects/graphql/queries.ts:260]: It was already removed, but Chuck forgot to remove it here

**Review by andrewwallacespeckle — DISMISSED** [review]

**Comment by andrewwallacespeckle** [comment]
Left a few minor comments, nothing that blocks the merge. 

**I only reviewed the frontend work.**

Nice work, a lot of major changes here in a short period of time!

**Review by andrewwallacespeckle — DISMISSED** [review]

**Review by benjaminvo — APPROVED** [review]

**Review by andrewwallacespeckle — APPROVED** [review]

**Pull request merged**